chart/openfaas/values.yaml

```diff
@@ -51,7 +51,7 @@ operator:
 
 # monitoring and auto-scaling components
 prometheus:
-  image: prom/prometheus:v2.3.1
+  image: prom/prometheus:v2.7.1
```

Comment:
@rdimitrov looks like we should use `v2.7.1` for Docker Swarm, we used `2.7.0` which may be my fault.